cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java

```diff
@@ -761,6 +762,8 @@ public String getDisplayName() {
           return "timestamp in microseconds";
         case DECIMAL:
           return String.format("decimal with precision %d and scale %d", precision, scale);
+        case DATETIME:
+          return String.format("datetime in ISO-8601 format without offset");
```

Comment:
should we call this `datetime in ISO-8601 format without timezone`? Since this will mostly be used for user facing error messages?